sdcm/cluster.py

```diff
@@ -1448,7 +1448,7 @@ def stop_scylla_jmx(self, verify_up=False, verify_down=True, timeout=300):
     @log_run_info
     def stop_scylla(self, verify_up=False, verify_down=True, timeout=300):
         self.stop_scylla_server(verify_up=verify_up, verify_down=verify_down, timeout=timeout)
-        self.stop_scylla_jmx(verify_up=verify_up, verify_down=verify_down, timeout=timeout)
+        self.stop_scylla_jmx(verify_up=False, verify_down=verify_down, timeout=timeout)
```

Comment:
Same question as above

Reply:
My comment (https://github.com/scylladb/scylla-cluster-tests/pull/807/commits/f505bd2ffa8a0342a3e18089730ba3ddd554587a#r259655566) is about stop_scylla()